src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -29,4 +34,50 @@ public static String convertToJsonUsingGson(Object object) {
     Gson gson = new Gson();
     return gson.toJson(object);
   }
+
+  /*
+   * Class that returns the ID of a logged in user.
```

Comment:
nit: This is a function and not a class, typically for functions it's better to write Javadoc that starts with a verb explaining what the method actually does.

```suggestion
   * Returns the ID of a logged in user.
```

Reply:
Also would be good to mention the fail state here - "If the user is not logged in or if no user ID is found, returns -1." The purpose of Javadoc is so that other developers don't have to spend time reading your actual code - they can just read the docs/comments and understand how the logic works!

Reply:
Don't know why I said it was a class, haha. Noted and done!